nodes/src/nodes/preprocessor_langchain/langchain.py

```
@@ -74,6 +74,9 @@ def _parseSeparators(self, user_input: str):
         """
         Parse a quoted, comma-separated string into a list of actual separator strings.
         """
+        _MAX_SEPARATOR_INPUT_LEN = 1024
+        if len(user_input) > _MAX_SEPARATOR_INPUT_LEN:
+            raise ValueError(f'Separator input too long ({len(user_input)} chars, max {_MAX_SEPARATOR_INPUT_LEN})')
```

Ruff (0.15.7)

[warning] Avoid specifying long messages outside the exception class

(TRY003)